src/addons/Portal/Portal.js

```diff
@@ -355,7 +355,7 @@ class Portal extends Component {
     document.addEventListener('keydown', this.handleEscape)
 
     const { onMount } = this.props
-    if (onMount) onMount()
+    if (onMount) onMount(undefined, this.props)
```

Comment:
@levithomason - this doesn't happen via a user event, so there's no event to pass through. Should we:
- pass undefined, since there's technically no event
- pass an empty object so if people try to access properties like `e.currentTarget` or something it won't die.